lib/puppet-lint/plugins/check_parameter_documentation.rb

```diff
@@ -1,5 +1,7 @@
 PuppetLint.new_check(:parameter_documentation) do
   def check
+    check_private = PuppetLint.configuration.docs_check_private_params || false
```

Comment:
`nil` and `false` evaluate to the same
```suggestion
    check_private = PuppetLint.configuration.docs_check_private_params
```